app/models/target.rb

```diff
@@ -0,0 +1,8 @@
+class Target < ApplicationRecord
+  belongs_to :topic
+  validates :tittle, presence: true
+  validates :radius, presence: true
+  validates :lat, presence: true
+  validates :lng, presence: true
+  validates :topic_id, presence: true
```

Comment:
you can simplify on only one line:

`validates :title, :radius, :lat, :lng, :topic_id, presence: true`

Reply:
wow, looks more clean. thanks!

Reply:
What if you also validate also that `radius` is greatear than 0?